src/patternfly/components/Divider/divider.scss

```diff
@@ -1,16 +1,40 @@
 $pf-c-divider--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl", "2xl");
 $pf-c-divider--spacer-map: build-spacer-map("none", "xs", "sm", "md", "lg", "xl", "2xl", "3xl");
 
-.pf-c-divider {
-  // update during breaking change
-  --pf-c-divider--Height: var(--pf-global--BorderWidth--sm);
-  --pf-c-divider--BackgroundColor: var(--pf-global--BorderColor--100);
-  --pf-c-divider--after--Height: var(--pf-c-divider--Height);
-  --pf-c-divider--after--BackgroundColor: var(--pf-c-divider--BackgroundColor);
+@mixin pf-c-divider--m-horizontal {
+  --pf-c-divider--Height: var(--pf-c-divider--Height--base);
+  --pf-c-divider--BackgroundColor: var(--pf-c-divider--BackgroundColor--base);
+  --pf-c-divider--after--Height: var(--pf-c-divider--after--Height--base);
+  --pf-c-divider--after--BackgroundColor: var(--pf-c-divider--BackgroundColor--base);
+
+  display: var(--pf-c-divider--Display--base);
+  flex-direction: var(--pf-c-divider--FlexDirection--base);
+}
+
+@mixin pf-c-divider--m-vertical {
+  display: var(--pf-c-divider--m-vertical--Display);
+  flex-direction: var(--pf-c-divider--m-vertical--FlexDirection);
+  width: var(--pf-c-divider--m-vertical--Width);
+  height: var(--pf-c-divider--m-vertical--Height);
+  min-height: var(--pf-c-divider--m-vertical--MinHeight);
+  max-height: var(--pf-c-divider--m-vertical--MaxHeight);
+
+  &::after {
+    flex-basis: calc(var(--pf-c-divider--m-vertical--after--FlexBasis) - var(--pf-c-divider--after--Inset));
+    width: var(--pf-c-divider--m-vertical--after--Width);
+  }
+}
 
-  // end - update during breaking change
 
-  --pf-c-divider--Display: flex;
```

Comment:
Looks like we lost the ability to set this and change the divider display, which would be a breaking change. We can work on it tomorrow and try and get it in.

Reply:
Thank you @mcoker , that sounds like a good plan!